### PR TITLE
Adapt data structure of conventional pp to scalar data template

### PR DESCRIPTION
**Pull request by juliusmeier**
The prepared data of conventional power plant data is adapted to the template of scalar data.
Closes issue #48

### CODE REVIEW

**Review by jnnr — CHANGES_REQUESTED** [review]
Thanks for your work! I produced the resource and found the following:

* name, type should be None
* unit should be None for chp and efficiency
* map carrier and tech from OPSD terminology to oemoflex's carriers and techs
* reference: add the items to a bibtex (references_input_data.bib, separate from the bibtex that is already included in the docs) and state the citation key here.

**Comment by juliusmeier** [comment]
- [ ] map carrier and tech from OPSD terminology to oemoflex's carriers and techs

**Review by juliusmeier — COMMENTED** [review]
Done

**Review by juliusmeier — COMMENTED** [review]
Done

File: scripts/prepare_conv_pp.py [hunk 1]
```diff
@@ -45,7 +48,7 @@
 
     regions_nuts3_de = geo.load_regions_file(in_path2)
 
-    b3_regions_yaml = open(in_path3, "r")
+    b3_regions_yaml = open(in_path3, "r", encoding="utf-8")
     b3_regions_content = yaml.load(b3_regions_yaml, Loader=yaml.FullLoader)
```
Comment (jnnr) [inline scripts/prepare_conv_pp.py:52]:
```suggestion
    with open(in_path3, "r", encoding="utf-8") as b3_regions_yaml:
        b3_regions_content = yaml.load(b3_regions_yaml, Loader=yaml.FullLoader)
```

**Pull request merged**